env.local.php

```diff
@@ -1,47 +0,0 @@
-<?php
```

Comment:
By convention, the sample configuration should not be renamed; rather, _copied_ to the location for configuration. The actual (not sample) configuration location should be in `.gitignore` to ensure that real config settings, including passwords, are never pushed to a repository.